ush/run_jedi_exe.py

```diff
@@ -74,6 +74,8 @@ def run_jedi_exe(yamlconfig):
         'CASE_ENKF': executable_subconfig.get('case_enkf', executable_subconfig['case']),
         'DOHYBVAR': executable_subconfig.get('dohybvar', False),
         'LEVS': str(executable_subconfig['levs']),
+        'forecast_steps': executable_subconfig.get('forecast_steps', 'PT6H'),
+        'BKG_TSTEP': executable_subconfig.get('forecast_step', 'PT6H'),
```

Comment:
It is not intuitively clear to me the purpose of adding the configurable `forecast_step` here. I assumed that `forecast_steps` is a list of times with forecasts available and `forecast_step` just indicates the temporal frequency of the list, yes? But isn't the frequency already indicated by `forecast_steps`? Correct me if I'm wrong ;)

Reply:
No you're not wrong, you're just noticing my laziness :-) Do you think it should compute BKG_TSTEP from the forecast_steps now? Or is it fine to let the user have to define this? Also, then, which should be defined and which derived? PT6H -> PT6H but PT3H -> PT3H PT6H PT9H and PT1H -> PT3H PT4H...

Reply:
I think it could be written so one can supply either one of the parameters. If only `forecast_step` is specified, then calculate the list of `forecast_steps`, if `forecast_steps` is specified, then use it directly. I can see that in some cases when the forecast times are not equally spaced, one can only specify with `forecast_steps`, but in normal situation `forecast_step` is more straightforward. So I would suggest to let the program check `forecast_steps` first, if not, then check `forecast_step` and calculate the list of times from it assuming equal spacing.

Reply:
They actually do have to be equally spaced. The 'forecast steps' are used for R2D2, and that only matters based on what is stored in the database. The 'step' is used in the 4D H(x) to know when to look for the next background, so they really do have to be consistent.

Reply:
OK, if the two have to be consistent, then only one of them needs/should to be specified in yaml. The code can be responsible for deriving one from another. If there are two configurable options, what if the inputs are inconsistent? Then you'll need to write additional logic to handle that. As a user of GDASApp, I have a slight preference for having `forecast step` in yaml as it is easier to specify.

Reply:
I agree, making `forecast step` support either `PT1H`, `PT3H`, or `PT6H` is the most logical for the user. It's what GSI takes right now I think (nhrs_assim or something like that = 1,3,6). I'll make this change today and push a commit.